docs/docs/getting-started/04-examples/02-editor-example.md

```diff
@@ -15,88 +15,94 @@ on the page to expand either the code editor or graph view to full screen.
-[auto lifecycle events](../../concepts/05-building-blocks/03-components.md#auto-lifecycle-events).
+"auto-activate" toggle switches can be used to enable or disable
+[auto-lifecycle events](../../concepts/05-building-blocks/03-components.md#auto-lifecycle-events).
 
 When both toggle switches are enabled, the YAML code will show the corresponding events for the componnts:
 
 ```yaml
     events:
-      is_unconfigured:
-        lifecycle: configure
-      is_inactive:
-        lifecycle: activate
+      transitions:
+        on_load:
+          lifecycle:
+            component: timer
+            transition: configure
+        on_configure:
+          lifecycle:
+            component: timer
+            transition: activate
 ```
 
-Disabling one or both auto event switches will also remove the corresponding events from the YAML, while enabling the
-switch will regenerate the event in the YAML.
+Disabling one or both auto-lifecycle event switches will also remove the corresponding events from the YAML, while
+enabling the switch will regenerate the event in the YAML.
 
-<!-- TODO: Replace the following sections with adding interactive buttons 
+## Setting component parameters
 
-Show how we can:
-- go to the sidebar
-- click on add trigger button
-- drag to reposition
-- create an event edge to the timer
-- choose the event type (e.g. deactivate)
-- rename the button
+Component parameters can be edited directly in the settings. Try changing the value of the Timeout parameter to a
+different one and look for the corresponding change in the code editor. Then press play on the application and observe
+that the timer now has a different duration before triggering the "Is timed out" predicate.
 
-Repeat for a second button to show manual deactivation / activation
+## Adding and deleting elements
 
-Then say: now that we learned to add buttons and create event edges, try to do the same for components. As an exercise,
-search for and add timer components, set thea auto-lifecycle events, and connect the transition edges to recreate the
-example from scratch.
--->
+Let's now make the example a bit more interesting by adding more elements to the graph using the sidebar menu; press the
+(+) button in the corner of the graph to open the sidebar. At the top, you will see special entries
+"Hardware Interface", "Trigger Events Button", "Sequence" and "Condition", for which a dedicated example is included
+in later sections. These are followed by a list of components with a name and brief description.
+
+Trigger Events Buttons are interactive elements that can be used to trigger events through mouse clicks and interact
+with the flow of the application. Click on the Trigger Events Button in the sidebar to add such an element to the
+application. A new button should appear on the graph and in the code.
+
+The sidebar menu can be closed by clicking anywhere in the graph. Drag on the button to change its position in the
```

Comment:
Personal pref: omit "on" in "drag on" line 66

Reply:
I think that's a leftover from a previous sentence yes